dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py

```diff
@@ -69,15 +70,25 @@ def __init__(
     ):
         super().__init__(name, TaskType.SQL, *args, **kwargs)
         self.sql = sql
+        self.param_sql_type = sql_type
         self.datasource_name = datasource_name
         self.pre_statements = pre_statements or []
         self.post_statements = post_statements or []
         self.display_rows = display_rows
 
     @property
     def sql_type(self) -> int:
+        """If sql type is specified by parameter"""
```

Comment:
you write a wrong docstring for python, you should write in 

```py
Judgement sql type, it will return the SQL type for type `SELECT` or `NOT_SELECT`.

If parameter :param:`param_sql_type` dot not specific, will use regexp to check
which type of the SQL is. But if parameter :param:`param_sql_type` is specific
will use the parameter overwrites the regexp way
```

Reply:
and you should remove L87